docs/src/pages/getting-started/astro.mdx

```diff
@@ -1 +1,229 @@
-# Astro (Coming Soon!)
+import { Callout, Steps, Tab, Tabs } from "nextra-theme-docs";
+
+import EnvSetup from "./_shared/env-setup.mdx";
+import TailwindSetup from "./_shared/tailwind-setup.mdx";
+
+# Astro Setup
+
+You can use Astro with any of the frontend frameworks we support. This guide
+will walk you through setting it up using React, but the steps will be near
+identical for other frameworks too.
+
+<Callout type="info">
```

Comment:
Not in love with this. It's fine but I'd love for us to find a better place to visibly include "examples" in future

Reply:
agree - but let's do that in another PR where we deal with that explicitely for all doc/example combos